backend/src/prisma/cm/prisma.cm.module.ts

```diff
@@ -0,0 +1,15 @@
+import { Module } from "@nestjs/common";
+import { CaseManagementPrismaService } from "./prisma.cm.service";
+//Ignoring Sonar Warning on the line below since we control the prisma client.
+import { PrismaClient } from "../../../node_modules/.prisma/case_management"; // NOSONAR
```

Comment:
This is generated every time by our pipeline or when a dev switches branches? Since we exclude node_modules, I'm wondering if we'll have any issues with leaving it there